packages/projects-docs/pages/learn/environment/vm.mdx

```diff
@@ -38,25 +36,36 @@ Note that, the first time you import a project to CodeSandbox, it can take longe
 
 ## Persistence
 
-Everything you save in `/project` is guaranteed to be persisted between reboots, hibernations and forks. Every file change outside of `/project` will usually be persisted, but there is a chance that those changes will be cleared.
+We use [Dev Containers](https://codesandbox.io/docs/learn/environment/devcontainers) by default for all repositories and Devboxes. Your project is under `/workspaces/workspace`, and your home folder is under `/root`. Under the hood, the paths are in `/project`.
+
+<Callout>The workspace folder is not always `/workspaces/workspace`; it is dependent on the Dev Container configuration.</Callout>
+
+Everything in `/workspaces/workspace` is guaranteed to be persisted between reboots, hibernations and forks. We persist the workspace folder and the Docker engine state, like images, volumes, etc.
+
+Every file change outside of `/workspaces/workspace` will usually be persisted, but there is a chance that those changes will be cleared.
+
+Your project folder lives in `/workspaces/workspace/<repo-name>`; your home folder lives in `/workspaces/workspace/home/<username>`; your home folder is inaccessible to other users.
+
+If your branch or Devbox has not been accessed for a certain period (explained below), we delete the contents of `/workspaces/workspace`. This means that the next time you start this branch again, we will reinitialize `/workspaces/workspace` by doing a fresh clone.
```

Comment:
We delete the contents of the workspace, but take a backup of uncommitted changes for repos and all code in devboxes (other than the ones in `.gitignore`). Maybe we can merge this line and the next one.